docs/config/extras.md

```diff
@@ -103,6 +103,8 @@ with the intent of making these the default behavior in Stencil v5. When set to
 - Slot element loses its parent reference and disappears when its parent is rendered conditionally [(#3913)](https://github.com/ionic-team/stencil/issues/3913) (since v4.8.2)
 - Failed to execute 'removeChild' on 'Node' [(#3278)](https://github.com/ionic-team/stencil/issues/3278) (since v4.9.0)
 - React fails to manage children in Stencil slot [(#2259)](https://github.com/ionic-team/stencil/issues/2259) (since v4.9.0)
+<!-- TODO confirm fix version (propagate if necessary) -->
```

Comment:
I'll make sure this is accurate before the next release. Just not sure if that'll be a minor or patch release yet